CMakeLists.txt

```diff
@@ -83,9 +144,40 @@ set (MOD_VCF_SOURCES
         src/vcf/validate_optional_policy.cpp
         src/vcf/validator.cpp
         )
-add_library(mod_vcf ${MOD_VCF_SOURCES})
-add_dependencies(mod_vcf mod_odb)
-
+add_library(mod_vcf STATIC ${MOD_VCF_SOURCES})
+add_dependencies(mod_vcf mod_odb) # works even without this if we add mod_vcf as target_link_libraries
+target_link_libraries(mod_vcf ${THIRD_PARTY_LIBRARIES} mod_odb)
+
+
+
+# Libraries to link with executables
```

Comment:
Again writing it as `if (MAC) else` here would make things more clear and less redundant, because right now the libraries are set twice in MacOS.

Reply:
isn't it like reassigning the variable like done in python or bash ? I am OK with if-else way also. it looks more standardised way :)